### PR TITLE
fix: code inspection issues

### PR DESCRIPTION
**Pull request by gidomanders**
Removed redundant unit of measure.
Removed redundant local variables.
Removed explicit type argument any.
Renamed birthdate to date of birth.
Changed invalid CSS property value (font-display used where font-style
was meant to be used).
Changed all words with a typo (messed up character order or missing
character).
Closed tags with empty body correctly.
Simplified expressions.
Fixed parameter described in jsdoc that does not appear in function
signature.
Fixed value assigned is never used warning.
Ignored missing await for async function calls.
Ignored variable might not have been initialized warning.
Ignored const reassignment warning.

### CODE REVIEW

File: src/form/ValuePicker/ValuePicker.tsx [hunk 1]
```diff
@@ -220,7 +221,7 @@ export default function ValuePicker<T>(props: Props<T>) {
   // Have to do `props.multiple === true` here otherwise it does not
```
Comment (MrHus) [inline src/form/ValuePicker/ValuePicker.tsx]: If this indeed works the comment above can be removed

File: src/core/Button/Button.test.tsx [hunk 2]
```diff
@@ -201,7 +201,7 @@ describe('Component: Button', () => {
 
   describe('events', () => {
     let button: ShallowWrapper;
-    let onClickSpy: jest.Mock<any, any>;
+    let onClickSpy: jest.Mock;
```
Comment (MrHus) [inline src/core/Button/Button.test.tsx]: Good to see that this works now.

File: src/form/ModalPicker/single/ModalPickerSingle.test.tsx [hunk 3]
```diff
@@ -400,6 +401,7 @@ describe('Component: ModalPickerSingle', () => {
         const { promise, reject } = testUtils.rejectablePromise();
         addButtonCallbackSpy.mockReturnValueOnce(promise);
 
+        // noinspection ES6MissingAwait
```
Comment (MrHus) [inline src/form/ModalPicker/single/ModalPickerSingle.test.tsx]: What does the noinspection do?

File: src/form/ImageUpload/utils.ts [hunk 4]
```diff
@@ -60,7 +60,6 @@ export function cropToAvatarEditorConfig(crop: Crop): AvatarEditorConfig {
 export function calculateScale(scale: number, delta: number): number {
   const zoomFactor = Math.max(Math.abs(delta) / 1000, 0.1);
   const nextScale = delta > 0 ? scale - zoomFactor : scale + zoomFactor;
-  const clamped = Math.min(10, Math.max(1, nextScale));
 
-  return clamped;
+  return Math.min(10, Math.max(1, nextScale));
```
Comment (MrHus) [inline src/form/ImageUpload/utils.ts]: I don't like this change makes it less clear that this is a clamp...

File: src/form/ModalPicker/single/ModalPickerSingle.test.tsx (hunk 3 above)
Comment (gidomanders) [reply src/form/ModalPicker/single/ModalPickerSingle.test.tsx]: It is for IntelliJ (and possibly other IDEs) to ignore the warning that there's no `await` before the async function call. Not really a coding problem, but that 'issue' causes IntelliJ to warn you on every commit when you selected the inspect code before commit option. The feature is very good to make sure your code is not going to fail, but it bothers me if I have to check what issues it found and only those come up.

Comment (MrHus) [reply src/form/ModalPicker/single/ModalPickerSingle.test.tsx]: Please remove them, I don't want to force people to use Intellij.

File: src/form/ImageUpload/ImageUpload.test.tsx [hunk 5]
```diff
@@ -68,6 +68,7 @@ describe('Component: ImageUpload', () => {
       // @ts-ignore
       const imgUpload = new ImageUpload();
 
+      // noinspection JSConstantReassignment
```
Comment (MrHus) [inline src/form/ImageUpload/ImageUpload.test.tsx]: Remove all noinspections

**Pull request merged**